docs/express.md

```diff
@@ -20,22 +20,32 @@ import "@aikidosec/firewall";
 // ...
 ```
 
-That's it! Your app is now protected by Zen.
-
-If you want to see a full example, check our [express sample app](../sample-apps/express-mongodb).
-
 ## Blocking mode
 
-By default, the firewall will run in non-blocking mode. When it detects an attack, the attack will be reported to Aikido and continue executing the call.
+By default, the firewall will run in non-blocking mode. When it detects an attack, the attack will be reported to Aikido if the environment variable `AIKIDO_TOKEN` is set and continue executing the call.
 
-You can enable blocking mode by setting the environment variable `AIKIDO_BLOCKING` to `true`:
+You can enable blocking mode by setting the environment variable `AIKIDO_BLOCK` to `true`:
 
 ```sh
-AIKIDO_BLOCKING=true node app.js
+AIKIDO_BLOCK=true node app.js
 ```
 
 It's recommended to enable this on your staging environment for a considerable amount of time before enabling it on your production environment (e.g. one week).
 
+## Rate limiting and user blocking
+
+If you want to add the rate limiting feature to your app, modify your code like this:
+
+```js
+const Zen = require("@aikidosec/firewall");
+
+const app = express();
```

Comment:
Let's include `setUser(...)` in an example middleware?

Ad a route under `Zen.setupExpressIntegration` to show the order?